catalogue/webapp/components/SearchPageLayout/SearchNavigation.tsx

```diff
@@ -78,16 +78,31 @@ const SearchNavigation: FunctionComponent<SearchNavigationProps> = ({
       ? getUrlQueryFromSortValue(sortOptionValue)
       : undefined;
 
-    const link = linkResolver({
-      params: {
-        ...formValues,
-        ...(urlFormattedSort && {
-          sort: urlFormattedSort.sort,
-          sortOrder: urlFormattedSort.sortOrder,
-        }),
-      },
-      pathname: router.pathname,
-    });
+    let link;
+    if (queryValue !== inputValue) {
+      // strip all filters since the search query has changed
```

Comment:
I'm not sure this is right – do we really want to drop all filters when the search query changes? I'd think we want to keep all the filters/query params and just drop the page number.

Reply:
change made